### PR TITLE
Fixes issues due to which shipping rates and taxes were not being stored in the sales order

### PR DESCRIPTION
**Pull request by neilLasrado**

### CODE REVIEW

File: erpnext_foxycart_integration/api.py [hunk 1]
```diff
@@ -100,6 +100,22 @@ def make_sales_order(customer, foxycart_data, foxycart_settings):
 			"rate": item.get("product_price")
 		})
 	sales_order.set("items", sales_items)
+	taxes = []
```
Comment (vjFaLk) [inline erpnext_foxycart_integration/api.py:104]: Why?

File: erpnext_foxycart_integration/api.py [hunk 2]
```diff
@@ -100,6 +100,22 @@ def make_sales_order(customer, foxycart_data, foxycart_settings):
...
+		"charge_type": "Actual",
+		"account_head": foxycart_settings.shipping_account_head,
+		"description": foxycart_data.get("shipto_shipping_service_description"),
+		"tax_amount": cint(foxy_data.get("shipping_total"))
+	})
+	if cint(foxy_data.get("tax_total")) > 0:
+		taxes.append({
+			"charge_type": "Actual",
+			"account_head": foxycart_settings.tax_account_head,
+			"description": "Tax",
```
Comment (vjFaLk) [inline erpnext_foxycart_integration/api.py]: Not a tax?

**Review by vjFaLk — DISMISSED** [review]
Maybe move some of the hardcoded values into the Settings?

File: erpnext_foxycart_integration/api.py [hunk 3]
```diff
@@ -100,6 +100,22 @@ def make_sales_order(customer, foxycart_data, foxycart_settings):
 			"rate": item.get("product_price")
 		})
 	sales_order.set("items", sales_items)
+	taxes = []
+	taxes.append({
+		"charge_type": "Actual",
+		"account_head": foxycart_settings.shipping_account_head,
+		"description": foxycart_data.get("shipto_shipping_service_description"),
+		"tax_amount": cint(foxy_data.get("shipping_total"))
```
Comment (vjFaLk) [inline erpnext_foxycart_integration/api.py]: Did you even test this? `cint` is not even imported.

File: erpnext_foxycart_integration/api.py (hunk 1 above)
Comment (neilLasrado) [reply erpnext_foxycart_integration/api.py:104]: because the sky is so high! ;-p

File: erpnext_foxycart_integration/config/desktop.py [hunk 4]
```diff
@@ -9,6 +9,6 @@ def get_data():
 			"color": "red",
 			"icon": "octicon octicon-radio-tower",
 			"type": "module",
-			"label": _("ERPNext FoxyCart integration")
+			"label": _("FoxyCart integration")
```
Comment (vjFaLk) [inline erpnext_foxycart_integration/config/desktop.py]: Change to "Foxycart Integration"

**Review by vjFaLk — APPROVED** [review]

**Pull request merged**